config/longrun_configs/longrun_aquaplanet_dyamond.yml

```diff
@@ -22,3 +21,13 @@ surface_setup: "DefaultMoninObukhov"
 vert_diff: "DecayWithHeightDiffusion"
 precip_model: "0M"
 toml: [toml/longrun_aquaplanet.toml]
+output_default_diagnostics: true
+netcdf_output_at_levels: true
+netcdf_interpolation_num_points: [2880, 1440, 63] 
```

Comment:
The numbers are too large for the h_elem 60

Reply:
These were from my template for the high-res runs; @sriharshakandala I'd change them to [720, 360, 63] for the 60 elem buildkite job.